python/cudf/cudf/pandas/_wrappers/numpy.py

```diff
@@ -129,6 +129,19 @@ def wrap_ndarray(cls, arr: cupy.ndarray | numpy.ndarray, constructor):
     },
 )
 
+
+flatiter = make_final_proxy_type(
+    "flatiter",
+    cupy._indexing.iterate.flatiter,
```

Comment:
Can we not use `cupy.flatiter` directly?

Reply:
We can, thanks!